### ISSUE TITLE
DDI Modelisation

### ISSUE THREAD

**Issue by laurentC35** [InseeFr/Bowie#81]
Issue to follow DDI modelisation in progress

- [ ] https://github.com/InseeFr/Bowie/issues/80
- [ ] https://github.com/InseeFr/Bowie/issues/78
- [ ] https://github.com/InseeFr/Bowie/issues/77
- [ ] https://github.com/InseeFr/Bowie/issues/76
- [ ] https://github.com/InseeFr/Bowie/issues/75

**Comment by romaintailhurat** [comment]
I dont think this is the intent of the [`QuestionBlock`](https://ddialliance.github.io/ddimodel-web/DDI-L-3.3/item-types/QuestionBlock/) element: 

"_A QuestionBlock is a specific structure used in educational and other types of testing where an object (Stimulus Material) is provided and a set of questions are asked regarding the object. The QuestionBlock generally has related QuestionBlocks that measure similar skills or aptitudes and is used randomly within a set of questionnaires to create multiple versions of a single questionnaire that can be used with large groups for testing purposes._"

**Comment by BulotF** [comment]
Maybe a [QuestionGroup ](https://ddialliance.github.io/ddimodel-web/DDI-L-3.3/item-types/QuestionGroup/) would be fine....

Contains a group of Questions, which may be ordered or hierarchical. In addition to the name, label, and description of the group, the structure allows for defining the type of group using an optional controlled vocabulary, a reference to a defining universe or concept for the group, a subject classification for the group, and a listing of Questions and QuestionGroups in any order.
